scheduled-list.js

```diff
@@ -522,13 +556,14 @@
   };
 
   exports.ScheduledList = list;
+  exports.ScheduledList.scheduler = schedule;
 
   /* Internals
      ========= */
 
-  var debug = false;
+  var debug2 = false;
   function log(str) {
-    if (!debug) {
+    if (!debug2) {
       return;
```

Comment:
let's switch `log()` instances to `debug()` and get rid of this part :)
We just need a way to early return in `debugViewport` maybe
`if (debug.name !== 'log') { return; }`